docs/synthetics/ci-cd-integration.md

```diff
@@ -122,6 +122,631 @@ AllLocations    276 us-west-1   passed  3/3 Passed  https://apps.sematext.com/ui
+| | | RESPONSE_CODE | Response Code | Only HTTP monitor |
+| | | ERROR | Error | |
+| | | SSL_CERT_EXPIRY | SSL Certificate Expiry (days) | *Only > operator available |
+| key | STRING | -- | Alert condition field | Only for the following: |
+| | | synthetics.http.time.dns | DNS time (ms) | Metric |
+| | | synthetics.http.time.connect | Connect time (ms) | Metric |
+| | | synthetics.http.time.tls | TLS time (ms) | Metric |
+| | | synthetics.http.time.firstbyte | Time to first byte (ms) | Metric |
+| | | synthetics.http.time.download | Download time (ms) | Metric |
+| | | synthetics.browser.transfer.size  | Bytes Transferred (Bytes) | Metric - Only browser monitor |
+| | | synthetics.browser.request.count | Request Count | Metric - Only browser monitor |
+| | | synthetics.time.response | Response time (ms) | Metric |
+| | | synthetics.http.response.size | Response size (bytes) | Metric |
+| | | User-defined | JSON path | Response Body JSON |
+| | | User-defined | Header name | Response Header |
+| operator | STRING | -- | Operator type | YES |
+| | | contains | Contains | *See [API Reference: operator](#api-reference-operator)  |
+| | | does not contain | Does Not Contain | *See [API Reference: operator](#api-reference-operator) |
+| | | > | Greater Than | *See [API Reference: operator](#api-reference-operator) |
+| | | < | Less Than | *See [API Reference: operator](#api-reference-operator)|
+| | | = | Equal To | *See [API Reference: operator](#api-reference-operator) |
+| | | !=  | Not Equal To | *See [API Reference: operator](#api-reference-operator) |
+| | | Starts With | Starts With | *See [API Reference: operator](#api-reference-operator) |
+| value | STRING | User-defined | Rule value | YES |
+| enabled | BOOLEAN | -- | Enable/Disable the line | YES |
+| | | true | Line enabled | |
+| | | false | Line disabled | |
+
+### API Reference: operator
+
+**HTTP Monitor**
+| Operator | Alert Condition Availability
+| --- | --- |
+| Contains | Response Body |
+| | Response Body JSON |
+| | Response Header |
+| | Response Code |
+| | Error |
+| Does Not Contain | Metric |
+| | Response Body |
+| | Response Body JSON |
+| | Response Header |
+| | Response Code |
+| | Error |
+| > | Metric |
+| | Response Body JSON |
+| | Response Code |
+| | SSL Certificate Expiry (days) |
+| < | Metric |
+| | Response Body JSON |
+| | Response Code |
+| = | Metric |
+| | Response Body |
+| | Response Body JSON |
+| | Response Header |
+| | Response Code |
+| | Error |
+| != | Metric |
+| | Response Body |
+| | Response Body JSON |
+| | Response Header |
+| | Response Code |
+| | Error |
+| Starts With | Response Body |
+| | Response Body JSON |
+| | Response Header |
+| | Response Code |
+| | Error |
+
+**Browser Monitor**
+| Operator | Alert Condition Availability
+| --- | --- |
+| Contains | Error |
+| Does Not Contain | Metric |
+| | Error |
+| > | Metric |
+| < | Metric |
+| = | Metric |
+| | Error |
+| != | Metric |
+| | Error |
+| Starts With | Error |
```

Comment:
For some reason these tables aren't rendering properly in the viewer I'm using, whereas all the other ones show up fine. Probably an issue on my end, but might be worth double checking.